probe-rs-tools/src/bin/probe-rs/cmd/cargo_embed/mod.rs

```diff
@@ -197,19 +196,6 @@ fn main_try(args: &[OsString], offset: UtcOffset) -> Result<()> {
     let (mut session, probe_options) = match probe_options.simple_attach(&lister) {
         Ok((session, probe_options)) => (session, probe_options),
 
-        Err(OperationError::MultipleProbesFound { list }) => {
```

Comment:
This goes away because `cargo embed` has no `--non-interactive` flag, so probe-rs interactively asks the user for a probe.